cmo/util.py

```diff
@@ -13,7 +13,7 @@
 #genomes = defaultdict(dict)
 #genomes['hg19']={"fasta":"/ifs/depot/assemblies/H.sapiens/hg19/hg19.fasta"}
 
-resource_file = os.getenv('CMO_RESOURCE_CONFIG', "/opt/common/CentOS_6-dev/cmo/cmo_resources.json")
+resource_file = os.getenv('CMO_RESOURCE_CONFIG', "/home/johnsoni/cmo/cmo/data/cmo_resources.json")
 json_config = json.load(open(resource_file))
 programs = json_config['programs']
 genomes = json_config['genomes']
```

Comment:
So... there's a bunch of these changes that I made to use my copies of the `cmo` and `roslin` repos. 

I guess this can't be a relative path, because after doing `setup.py install --user` this `util.py` file gets put in a directory separate from the original source. 

Perhaps this could be solved with environment variables, or a config file (maybe `config.ian.yaml`) where I have a separate variable for the `cmo_resources.json` file.

Reply:
@ionox0  lol--that IS an env variable.  if you define CMO_RESOURCE_CONFIG, it respects it.  you didn't need to change this line. the second variable is if it is undefined.

Reply:
Ohh haha. Wow right right of course. Will change back 👍